components/arm/universalrobots/ur5e_test.go

```diff
@@ -49,8 +49,8 @@ func testUR5eInverseKinematics(t *testing.T, pos spatialmath.Pose) {
 	m, err := referenceframe.UnmarshalModelJSON(ur5modeljson, "")
 	test.That(t, err, test.ShouldBeNil)
 	steps, err := motionplan.PlanFrameMotion(ctx, logger, pos, m, referenceframe.FloatsToInputs([]float64{0, 0, 0, 0, 0, 0}), nil, nil)
-
```

Comment:
Can we undo this now-non-existent change so that the git history doesn't get polluted?